etcdutl/etcdutl/printer.go

```diff
@@ -78,6 +80,16 @@ func makeDBStatusTable(ds snapshot.Status) (hdr []string, rows [][]string) {
 	return hdr, rows
 }
 
+func makeDBHashKVTable(ds snapshot.HashKV) (hdr []string, rows [][]string) {
+	hdr = []string{"hash", "hash revision", "compact revision"}
+	rows = append(rows, []string{
+		fmt.Sprintf("%x", ds.Hash),
```

Comment:
Let's keep it consistent with the output of `etcdctl endpoint hashkv`
```suggestion
		fmt.Sprint(ds.Hash),
```